### PR TITLE
Rework README and bootstrap directly into a running machine

### PR DESCRIPTION
**Pull request by bascht**
Refs #519

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,58 +27,57 @@ When you want to add an event to [hacken.in](http://hacken.in) please check with
...
 
 ```ruby
 config.vm.synced_folder ".", "/opt/hacken.in", type: 'rsync', rsync__args: ['-a']
 ```
 
-Afterwards local code changes need to be re-synced to the VM via `$ vagrant reload --provision`.
-
-If the Vagrant exits successfully, you can access your local machine under the URL [hacken.local](http://hacken.local). All the files you
-change in your local folder are synced to the virtual machines, so you can work on hacken.in right away. :wrench:
+Afterwards local code can to be re-synced to the VM via `$ vagrant rsync-auto` which you can run in the background .
```
Comment (moonglum) [inline README.md]: A wild space appears.

**Comment by moonglum** [comment]
Looking really good. Thanks :smile: One thing:

We have a paragraph about guard. Is it applicable to both of the setups? If so, how would you run guard in the vagrant setup – you would run it inside of the VM. Same goes for executing tests etc. I guess we should describe that, right? If someone has never used Vagrant before, they might be confused.

**Comment by cashewkernchen** [comment]
I would add the information about vagrant ssh for running rspec (which I stumpled upon), besides that I think it's comprehensible (:

**Comment by bascht** [comment]
@moonglum NFS mounts won't send inotify events, so Guard has to Poll. :sob:

**Comment by bascht** [comment]
@cashewkernchen The `script/bootstrap` drops into a shell with a hint on how to run the tests:
screenshot from 2015-01-16 11 02 44

Is this what you had in mind? (Also related to #520 – so users already find test data in the DB)

**Comment by moonglum** [comment]
@bascht Damn... :frowning:

**Comment by bitboxer** [comment]
Looking good! Merging noaaaww.

**Pull request merged**